### ISSUE TITLE
Pivot: underline not visible in High Contrast mode

### ISSUE THREAD

**Issue by pablonete** [OfficeDev/office-ui-fabric-react#1183]
### Bug Report

- __Package version(s)__: 1.7.2

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

#### Describe the issue:

In High Contrast mode, the underline of the selected Pivot doesn't show up.

In Windows, use LShift+LAlt+PrtSc to enable High Contrast mode, then browse with Edge to the samples page.

#### Actual behavior:

![image](https://cloud.githubusercontent.com/assets/579705/23606484/382cf126-0261-11e7-8900-8453475651bb.png)

#### Expected behavior:
Same underline than in standard mode:
![image](https://cloud.githubusercontent.com/assets/579705/23606474/30b1afcc-0261-11e7-9978-9cfb734e166d.png)

**Comment by phkuo** [comment]
The underline is provided using a background color, hence why it's disappearing in high contrast. Using a border or outline (even with color transparent) _should_ fix the issue.

**Comment by dzearing** [comment]
Resolved.
